src/js/shared/ErrorMessage.js

```diff
@@ -0,0 +1,24 @@
+let timeoutHandler = null;
+
+const clearError = () => {
+  const errorDiv = document.getElementById("errorDiv");
+
+  errorDiv.classList.remove("visible");
+  errorDiv.textContent = "";
+};
+
+export const ErrorMessage = errorMessage => {
+  const header = document.querySelector("header");
+
+  const errorDiv = document.createElement("div");
+  errorDiv.id = "errorDiv";
+  errorDiv.classList.add("alert-danger");
+
+  errorDiv.textContent = errorMessage;
+  errorDiv.classList.add("visible");
+
+  header.append(errorDiv);
```

Comment:
지금처럼 DOM 객체를 그대로 사용하면 사실 이 코드를 보는 사람 입장에서 머릿속에 구조를 그리기가 무척 어렵답니다 ㅠㅠ

```suggestion
  header.append(createElementByTemplate(`<div id="errorDiv" class="alert-danger visible">${errorMessage}</div>`));
```
그래서 이렇게 표현할 수 있도록 함수를 하나 만들어서 관리해주면 어떨까요?